Content.Server/Voting/VoteCommands.cs

```diff
@@ -92,6 +108,40 @@ public void Execute(IConsoleShell shell, string argStr, string[] args)
                 options.Options.Add((args[i], i));
             }
 
+            // Set up the webhook payload
+            var _gameTicker = _entitySystem.GetEntitySystem<GameTicker>();
+
+            var payload = new WebhookPayload()
+            {
+                AvatarUrl = _avatarUrl,
+                Username = _serverName,
+                Embeds = new List<Embed>
+                {
+                    new()
+                    {
+                        Title = $"{shell.Player}",
+                        Color = 13438992, // 2353993
+                        Description = options.Title,
+                        Footer = new EmbedFooter
+                        {
+                            Text = $"{_serverName} {_gameTicker.RoundId} {_gameTicker.RunLevel}",
+                        },
+
+                        Fields = new List<Field> {},
+                    },
+                },
+            };
+            
+            foreach (var voteOption in options.Options)
+            {
+                var NewVote = new Field() { Name = voteOption.text,  Value = "0"};
+                payload.Embeds[0].Fields.Add(NewVote);
+            }
+
+            Console.Write(JsonSerializer.Serialize(payload));
```

Comment:
Remove

Reply:
Yeah that was just me trying to figure out why it wasn't working, will remove once it is